build.sbt

```diff
@@ -63,11 +65,3 @@ publishTo <<= (version)(version =>
 resolvers += "Public Group" at nexusUrl + nexusPublicGroupPath
 
 resolvers += "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"
-
-enablePlugins(GitVersioning)
-
-git.gitTagToVersionNumber := { tag: String =>
-    Some(tag.replace("release/", ""))
-}
-
-git.uncommittedSignifier := None
```

Comment:
I removed the git-tag based versioning because it makes development difficult (you need to git-tag to have a new version but what about snapshots) and to use the same explicit and more obvious versioning scheme as we have in our other libraries.